harness/determined/pytorch/_pytorch_context.py

```diff
@@ -4,7 +4,7 @@
 from typing import Any, Callable, Dict, Iterator, List, Optional, Set, Tuple, Type, Union
 
 import torch
-import torch.nn as nn
+from torch import nn
```

Comment:
semantically these end up being slightly different: 
- you can do `from determined.core import CheckpointContext`, i.e. import a non-module attribute in the `core` module
- but you can't do `import determined.core.CheckpointContext as CheckpointContext`.

for importing modules, it's the same. if we only allow importing modules, this does not matter. looks irrelevant for us.

torch docs use `import torch.nn as nn` quite often, probably because they are not bound by the same style guide.